packages/cli/src/cmds/preview/preview.ts

```diff
@@ -557,6 +573,53 @@ export default class Preview extends YargsCommand {
     return ok(null);
   }
 
+  private async localPreviewMinimalApp(
+    workspaceFolder: string,
+    browser: constants.Browser,
+    browserArguments: string[] = []
+  ): Promise<Result<null, FxError>> {
+    const coreResult = await activate();
+    if (coreResult.isErr()) {
+      return err(coreResult.error);
+    }
+    const core = coreResult.value;
+
+    const inputs: Inputs = {
+      projectPath: workspaceFolder,
+      platform: Platform.CLI,
+      env: environmentManager.getLocalEnvName(),
```

Comment:
No need to check isConfigUnifyEnabled?

Reply:
No need, since minimal app is only supported (init/preview...) when `isConfigUnifyEnabled` is enabled.